tests/test_base.py

```diff
@@ -0,0 +1,54 @@
+from api import models
+from django.test import Client
+from django.contrib.auth import login
```

Comment:
Can you remove the unused imports? That includes these first three, `django.test.Client`, `rest_framework.test.force_authenticate`, and `GroupFactory`